config/src/main/resources/net/sourceforge/pmd/pmd-ui-dogfood-config.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<ruleset name="pmd-ui-dogfood"
+    xmlns="http://pmd.sourceforge.net/ruleset/2.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 http://pmd.sourceforge.net/ruleset_2_0_0.xsd">
+    <description>Rules specific to the UI module.</description>
+
+    <rule ref="net/sourceforge/pmd/pmd-dogfood-config.xml" />
+
+    <rule name="FxIdNamingConvention"
+          language="java"
+          since="6.5.0"
+          message="Injected FXML fields should be suffixed by their type, eg 'HelloButton'"
+          class="net.sourceforge.pmd.lang.rule.XPathRule" >
+        <description>
+            TODO
+        </description>
+        <priority>2</priority>
+        <properties>
+            <property name="version" value="2.0" />
+            <property name="xpath">
+                <value><![CDATA[
+
+//FieldDeclaration[../Annotation[pmd-java:typeIs('javafx.fxml.FXML')]]
```

Comment:
The namespace prefix has become required since pmd/pmd#1148. This should either be documented or fixed. I think using the prefix can clarify the rule, but that it would be better for readability to just use `pmd`.

Reply:
the namespace has always been required when using xpath 2. xpath 1 doesn't need it.

Even the old `typeof` function documented this:

https://github.com/Monits/pmd/blob/48861e1da28c642f0884b50ba204fc2caa506350/pmd-java/src/main/java/net/sourceforge/pmd/lang/java/xpath/TypeOfFunction.java#L76-L78